Gemfile

```diff
@@ -40,6 +40,12 @@ gem 'webpacker'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'httparty'
+
+gem 'pry-byebug'
```

Comment:
We already have `byebug` in the `development, test` group below.
So you can use that already. It should be pretty much the same as this one, so you can remove this.